api/src/pcapi/core/offers/offer_metadata.py

```diff
@@ -60,18 +65,19 @@ def _get_common_metadata_from_offer(offer: offers_models.Offer) -> Metadata:
 def _get_event_metadata_from_offer(offer: offers_models.Offer) -> Metadata:
     common_metadata = _get_common_metadata_from_offer(offer)
 
-    event_metadata = {
+    event_metadata: Metadata = {
         "@type": "Event",
-        "location": _get_metadata_from_venue(offer.venue),
     }
 
-    if offer.firstBeginningDatetime:
-        firstBeginningDatetime: datetime.datetime = offer.firstBeginningDatetime
-        event_metadata["startDate"] = firstBeginningDatetime.isoformat(timespec="minutes")
+    if not offer.isDigital:
+        event_metadata["location"] = _get_metadata_from_venue(offer.venue)
+        event_metadata["eventAttendanceMode"] = "OfflineEventAttendanceMode"
+    else:
+        event_metadata["eventAttendanceMode"] = "OnlineEventAttendanceMode"
```

Comment:
le `not` ajoute une petite compléxité à lire

```suggestion
    if offer.isDigital:
        event_metadata["eventAttendanceMode"] = "OnlineEventAttendanceMode"
    else:
        event_metadata["location"] = _get_metadata_from_venue(offer.venue)
        event_metadata["eventAttendanceMode"] = "OfflineEventAttendanceMode"
```